dashboard/test/factories/factories.rb

```diff
@@ -360,6 +362,7 @@
           credential_type: AuthenticationOption::EMAIL,
           authentication_id: user.hashed_email
         )
+        user.authentication_options << ao
```

Comment:
Is this necessary, given that `authentication_options` is a `has_many` reference and the `user` object is already set above? I don't object to it, I'm just curious if there's something that happens here (in which case I want to make sure to do this everywhere) or if it's a no-op.

Reply:
I think it's a no-op.  I don't remember why I did this now - somewhere in the middle of this work I was having trouble getting a test to pass and this did the trick, but I just removed this line and all the tests still pass.  I'll tear it out in a followup PR.